### ISSUE TITLE
Cursor is not hidden in fullscreen unless it is moved

### ISSUE THREAD

**Issue by TheFeelTrain** [jellyfin/jellyfin-media-player#183]
While in the player, the cursor does not get hidden unless you move it.

**To Reproduce**
1. Click play on any video
2. Do not touch and/or move the mouse
3. Press F to enter fullscreen
4. The menu elements hide after a couple seconds but the cursor will stay until it is moved

**Expected behavior**
The cursor should be hidden at the same time the menu is.

**Screenshots**
Movie playing in fullscreen.
![Screenshot_20211226_230432](https://user-images.githubusercontent.com/12601452/147444630-f3299f98-9723-4456-ba6c-935fc276b721.png)

**Desktop**
 - OS: Arch Linux (using KDE Plasma, not sure if it happens in other DEs)
 - Version: Happens in all versions, currently using 1.6.1
 - Installation Method: AUR

**Additional context**
I tested and it also happens in Plex Media Player, so it's a bug inherited from there. (F doesn't fullscreen in PMP so you need to click the button in the top right without moving the mouse afterwards)

**Comment by bsp-tree** [comment]
Just for the dev's awareness, I was not able to recreate this behavior on Windows.

**Comment by TheFeelTrain** [comment]
It could very well be just a KDE and/or Linux only bug. Here is also a short video of it happening.
You can see after I wiggle it, the mouse does disappear. So it's just something to do with switching into fullscreen.

https://user-images.githubusercontent.com/12601452/147847085-ae1b0c20-b49c-44f6-bf2e-b75b4f52e87e.mp4

**Comment by Rorschach1010** [comment]
I have this issue too. OS is EndeavourOS (Arch) with KDE. Happens with the AUR package as well as Flatpak.

**Comment by jameskimmel** [comment]
> Just for the dev's awareness, I was not able to recreate this behavior on Windows.

Same for macOS

**Comment by mijofa** [comment]
I'm seeing the same issue on Debian Bullseye, under both Wayland (with phoc) & X11 (with XFCE) while trying to setup a standalone jellyfin-media-player system. Bit to just have a pointer stuck in the top-left when there isn't even a mouse plugged in.

**Comment by Suzaru** [comment]
This is sadly still the case. I'm on MacOS and the mouse will not disappear.

**Comment by Rorschach1010** [comment]
yeah, still not fixed and sadly there is not much happening here. Last commit is from august last year.

**Comment by X5-176** [comment]
Not just Linux either this is happening on the Jellyfin Xbox app also, I don’t see why there is a cursor on the Xbox Jellyfin to begin with, it should be like the emby or plex app where you move from selection to selection with the controller.

**Comment by mijofa** [comment]
@X5-176, this is the desktop app, it's not available for Xbox. You're either using [jellyfin-uwp](https://github.com/jellyfin/jellyfin-uwp) or you're using the builtin web browser where Jellyfin doesn't really control the cursor display.

**Comment by Rainbowlemon** [comment]
This continues to be an issue for me on Linux running the non-flatpak version of JMP. (Previously had the flatpak and it had the same issue). I have to manually load up my media server in a VNC window from my other machine to move the mouse to the bottom right corner of the screen.

**Comment by reinduhr** [comment]
I hope this helps:

On my Pop!_OS 22.04 LTS (without moving the cursor) I opened a local video file with mpv in fullscreen and it immediately hid the cursor. I tried both clients from flatpak and I guess the other one is from apt (I used the Pop shop), they had the same result.
Jellyfin Media Player flatpak however does not hide the cursor automatically. Jellyfin only hides it if you move it a bit.

It would be a step in the right direction if this could be resolved. Thank you. ;)

**Comment by cgsmith** [comment]
Happens in Debian 12 as well but it stays even after wiggling. Apple MacBook Pro M4 - JellyFin App 1.6.1.0
